### PR TITLE
No space before ellipsis

### PR DESCRIPTION
**Pull request by ovari**
https://github.com/PhoenicisOrg/phoenicis/issues/1466

### CODE REVIEW

File: phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java [hunk 1]
```diff
@@ -156,7 +156,7 @@ public Void presentation(String programName, String programEditor, String applic
                 + tr("{0} will be installed in: {1}", programName, applicationUserRoot) + "<br><br>"
                 + tr("{0} is not responsible for anything that might happen as a result of using these scripts.",
                         applicationName)
-                + "<br><br>" + tr("Click \"Next\" to start.") + "</body>";
+                + "<br><br>" + tr("Click Next to start.") + "</body>";
```
Comment (madoar) [inline phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java]: Why did you remove the "" here?

Comment (ovari) [reply phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java]: It seems that the current convention is to not have quotation marks around buttons labels in descriptions.

https://transvision.mozfr.org/?recherche=Click+Next+to+&repo=gecko_strings&sourcelocale=en-US&locale=en-US&search_type=strings 

> ### 1.3 Welcome
> To continue, click Next.
> ### 1.9 LibreOffice Installation Completed
> Click Finish to exit the wizard.
https://www.libreoffice.org/get-help/install-howto/windows/
